SubscribeIT/ngDesk
# Fixed chat edit layout

Closes #353 

Test case 1
Name :- Personal details in chat edit layout
1. Login to ngdesk.
2. Setup chat widget.
3. Goto -> chat edit layout.
4. Observe the personal details.
5. Observe no error.

Test case 2
Name :- Technology details in chat edit layout
1. Login to ngdesk.
2. Setup chat widget.
3. Goto -> chat edit layout.
4. Observe the technology details.
5. Observe no error.

Screenshots :- 

![Screenshot from 2022-02-02 14-00-47](https://user-images.githubusercontent.com/89504119/152119162-01be1044-9266-4754-b586-7cf4871bfab2.png)
